pallets/linear-release/src/mock.rs

```diff
@@ -124,6 +97,9 @@ pub enum MockRuntimeHoldReason {
 	Reason,
 	Reason2,
 }
+impl VariantCount for MockRuntimeHoldReason {
```

Comment:
Can't we just derive `VariantCount` from the crate `variant_count`, which always gives us the correct number of variants at compile time?
Or is that trait necessary for HoldReason?

Reply:
I'm already using that crate for `AcceptedFundingAsset`

Reply:
Don't think this works for RuntimeHoldReason as there is a frame trait bound and this crate doesn't implement this specific trait as its frame specific.